adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/db/schema/ConnectedClientsSchema.java

```diff
@@ -63,7 +63,7 @@ public PreparedStatement connectionsByUser()
         return connectionsByUserStatement;
     }
 
-    private String statsStatement()
+    static String statsStatement()
```

Comment:
On a side note, those changes should ideally be brought up during the review of that patch and addressed there.